style.css

```diff
@@ -0,0 +1,116 @@
+* {
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+}
+
+body {
+	min-height: 100vh;
+
+	background: #222222;
+
+	display: flex;
+	justify-content: center;
+	align-self: center;
+}
+
+.container {
+	margin: 25px;
+
+	position: relative;
+
+	max-width: 100%;
+
+	display: grid;
+	grid-template-columns: repeat(auto-fill, minmax(350px, 1fr));
+	grid-template-rows: minmax(100px, auto);
+	grid-auto-flow: dense;
+	gap: 10px;
+}
+
+.container .box {
+	background: #333333;
+	padding: 20px;
+
+	font-size: 20px;
+	text-align: center;
+	color: #ffffff;
+
+	display: grid;
+	place-items: center;
+
+	transition: 0.5s;
+}
+
+.box:hover {
+	background: #e91e63;
+}
+
+.container .box img {
+	position: relative;
+
+	max-width: 100px;
+
+	margin-bottom: 10px;
+}
+
+.container .box:nth-child(1) {
+	grid-column: span 2;
+	grid-row: span 1;
+}
+
+.container .box:nth-child(2) {
+	grid-column: span 1;
+	grid-row: span 2;
+}
+
+.container .box:nth-child(4) {
+	grid-column: span 1;
+	grid-row: span 2;
+}
+
+.container .box:nth-child(5) {
+	grid-column: span 3;
+	grid-row: span 1;
+}
+
+@media all and (max-width: 1010px) {
+	.container {
+		grid-template-columns: repeat(auto-fill, minmax(50%, 1fr));
+		grid-template-rows: minmax(auto, auto);
+	}
+
+	.container .box {
+		grid-column: unset !important;
+		grid-row: unset !important;
+	}
+}
+
+@media all and (min-width: 1760px) {
+	.container {
+		grid-template-columns: 1fr 1fr 1fr 1fr;
+		grid-template-rows: 1fr 1fr 1fr;
+		max-height: 1500px;
+	}
+	/* .container .box {
+		grid-column: unset !important;
+		grid-row: unset !important;
+	} */
```

Comment:
Try to avoid commented code while pushing on GitHub

Reply:
done..